packages/main/src/TextArea.ts

```diff
@@ -474,6 +474,12 @@ class TextArea extends UI5Element implements IFormElement {
 		}
 
 		this.value = nativeTextArea.value;
+		const valueLength = this.value.length;
+
+		if (e.inputType === "insertFromPaste" && this.maxlength && valueLength > this.maxlength) {
```

Comment:
this.maxlength can go into a variable as it is used 3 times, but im also fine it stays like this.